test/browser/features/lib/browser.rb

```diff
@@ -36,8 +36,8 @@ def supports_fetch_keepalive?
       @version >= 66
 
     when "edge"
-      # support added in Edge 15
-      @version >= 15
+      # On Edge 80 keepalive doesn't work for cross-origin preflight requests 
+      @version >= 81
```

Comment:
I wonder if this should still be 15 and up but exclude this specific version, as this definition should really cover when this feature is actually supported?